BUILDING.md

```diff
@@ -759,7 +759,156 @@ as `deps/icu` (You'll have: `deps/icu/source/...`)
 
 ## Building Node.js with FIPS-compliant OpenSSL
 
-The current version of Node.js does not support FIPS.
+The current version of Node.js does not support FIPS when statically linking
+(the default) with OpenSSL 1.1.1 but for dynamically linking it is possible
+to enable FIPS using the configuration flag `--openssl-is-fips`.
+
+### Configuring and Building quictls/openssl for FIPS
```

Comment:
```suggestion
### Configuring and building quictls/openssl for FIPS
```

Reply:
Thanks Rich, I'll update this and I'm also going through this now and updating to remove some of the notes related to alpha-15 as alpha-16 is now available.